src/modules/admin/admin-tab-categories/AdminTabCategories.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+import { NavLink, useRouteMatch, useHistory } from 'react-router-dom';
+
+import { Button as MuiButton } from '@material-ui/core';
+import UnfoldMoreIcon from '@material-ui/icons/UnfoldMore';
+
+import { Button } from 'ui';
+
+import { Url } from 'utils';
+
+import csx from './AdminTabCategories.scss';
+
+export enum AdminTabCategory {
```

Comment:
You should avoid adding things like that in component files - there should be separate models.ts file in modules/admin/models - added to index.ts as export *  and you should import AdminTabCategory